packages/client/hmi-client/src/components/workflow/ops/interventions/tera-interventions-node.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+	<section>
+		<tera-operator-placeholder :operation-type="node.operationType" />
+		<Button
+			:label="isModelInputConnected ? 'Open' : 'Attach a model'"
+			@click="emit('open-drilldown')"
+			severity="secondary"
+			outlined
+			:disabled="!isModelInputConnected"
+		/>
+	</section>
+</template>
+
+<script setup lang="ts">
+import { computed, watch } from 'vue';
+import { WorkflowNode, WorkflowPortStatus } from '@/types/workflow';
+import Button from 'primevue/button';
+import TeraOperatorPlaceholder from '@/components/operator/tera-operator-placeholder.vue';
+import { cloneDeep } from 'lodash';
+import { InterventionsState } from './tera-interventions-operation';
+
+const emit = defineEmits(['open-drilldown', 'update-state']);
+const props = defineProps<{
+	node: WorkflowNode<InterventionsState>;
+}>();
+
+const modelInput = props.node.inputs.find((input) => input.type === 'modelId');
+const isModelInputConnected = computed(() => modelInput?.status === WorkflowPortStatus.CONNECTED);
+
+watch(
+	() => props.node.inputs,
+	() => {
+		const inputs = props.node.inputs;
+		const modelInputs = inputs.filter((input) => input.type === 'modelId');
```

Comment:
You have only declared a single "input type" in the operator's definition, if we do not expect to have additional input types down the line we can simplify a few things here. e.g. the filter is basically a no-op.

Reply:
true, changes have been made so that I get the `modelId` instead